### PR TITLE
Add how to add other language apart from Japanese

### PR DESCRIPTION
**Pull request by foxmean**
Anyway, Cristian said in discord server that this plugin only work for 1 language. So, should we mentioned about it too? (Not mention yet).

### CODE REVIEW

**Comment by cristian428** [comment]
I got Chinese audio files from forvo working through this add on but i had to move all the files to the same folder [forvo_files] where the japanese user folders also are. Especially since Chinese and even many other languages are now being supported by Yomitan, would it be possible to update this add on to have a separate source/folder for other languages like for chinese "zh forvo" or something like that? That way we can just specify in yomitan in the custom url json "language=zh" to only fetch the chinese files. Or at least can you add instructions for how to separate language folders for forvo in the add ons folder?  The way it is now, I need to specify all the chinese speakers with forvo usernames in the custom url.

File: README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ There is currently no way of using this on AnkiMobile (iOS).
         > **However, Opus is NOT compatible with AnkiMobile (iOS), Android 4, and AnkiWeb.**
         > If you use any of these, please use the 2nd option (MP3 audio) below.
         >
-        > -   **Download the files from [this torrent](https://nyaa.si/view/1681655)**.
+        > -   **For people who learn Japanese, download the files from [this torrent](https://nyaa.si/view/1681655)**.
```
Comment (MarvNC) [inline README.md:67]:
```suggestion
        > -   **For Japanese audio, download the files from [this torrent](https://nyaa.si/view/1681655)**.
```

File: README.md [hunk 2]
```diff
@@ -73,13 +73,22 @@ There is currently no way of using this on AnkiMobile (iOS).
         >     ```
         >
         >     </details>
+        >
+        > -   **For people who learn other language**, use the magnet link below and select to download _only_ zip file of the language under the folder Forvo_pronunciation > export > opus > [iso 639 code of language that you want to learn.zip](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes):
```
Comment (MarvNC) [inline README.md:77]:
```suggestion
        > -   **For audio of other languages**, use the magnet link below containing Forvo audio and select the zip file of the language you want under the folder `Forvo_pronunciation > export > opus > [iso 639 code of language that you want to learn.zip](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes)`:
```

File: README.md [hunk 3]
```diff
@@ -73,13 +73,22 @@ There is currently no way of using this on AnkiMobile (iOS).
...
+        >
+        >     </details>
         </details>
 
     1.  <details>
         <summary>MP3 audio (4.9 GiB)</summary>
 
         > Older and less efficient codec, but needed for compatibility with pretty much all devices.
-        > -   **Download the files from [this torrent](https://nyaa.si/view/1681654)**.
+        > -   **For people who learn Japanese, download the files from [this torrent](https://nyaa.si/view/1681654)**.
```
Comment (MarvNC) [inline README.md:91]:
```suggestion
        > -   **For Japanese audio, download the files from [this torrent](https://nyaa.si/view/1681654)**.
```

File: README.md [hunk 4]
```diff
@@ -88,14 +97,23 @@ There is currently no way of using this on AnkiMobile (iOS).
...
         >     </details>
+        >
+        > -   **For people who learn other language**, use the magnet link below and select to download _only_ zip file of the language under the folder Forvo_pronunciation > export > mp3 > [iso 639 code of language that you want to learn.zip](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes):
+        >     <details><summary>Magnet link</summary>
+        >
+        >     ```
+        >     magnet:?xt=urn:btih:255A1B495719DD3CE0CF8AA708FE6837AD9F57E1&tr=http%3A%2F%2Fbt2.t-ru.org%2Fann%3Fmagnet&dn=%D0%9D%D0%B0%D0%B1%D0%BE%D1%80%20%D0%BF%D1%80%D0%BE%D0%B8%D0%B7%D0%BD%D0%BE%D1%88%D0%B5%D0%BD%D0%B8%D0%B9%20%D0%BE%D1%82%20Forvo.com%20%D0%B4%D0%BB%D1%8F%20GoldenDict%20%D0%BD%D0%B0%20%D0%B2%D1%81%D0%B5%D1%85%20%D1%8F%D0%B7%D1%8B%D0%BA%D0%B0%D1%85
+        >     ```
+        >
+        >     </details>
```
Comment (MarvNC) [inline README.md:108]: Why is this repeated twice? Is the audio in the torrent mp3 or ogg?

File: README.md [hunk 5]
```diff
@@ -88,14 +97,23 @@ There is currently no way of using this on AnkiMobile (iOS).
...
         </details>
 
     If you have never downloaded from a torrent before, I highly recommend using the
     [qBittorrent](https://www.qbittorrent.org/) client.
 
-2. Extract the `.tar.xz` file.
-    * **Windows** users can use [7zip](https://7-zip.org/download.html). Note that 7zip users must extract the resulting `tar` file as well.
-    * **Linux and MacOS** users can use either the default GUI archive manager or the `tar -xf` command.
+2. Extract the `.tar.xz` (Japanese), or `.zip` file for other language.
```
Comment (MarvNC) [inline README.md:114]:
```suggestion
2. Extract the `.tar.xz` (Japanese), or `.zip` file for other languages.
```

File: README.md [hunk 6]
```diff
@@ -114,9 +132,9 @@ There is currently no way of using this on AnkiMobile (iOS).
 
         ![image](./img/view_files.png)
 
-    * Copy the entire `user_files` folder (from the second step) into this folder.
+    * For people who learn Japanese, copy the entire `user_files` folder (from the second step) into this folder.
```
Comment (MarvNC) [inline README.md:135]:
```suggestion
    * For Japanese, copy the entire `user_files` folder (from the second step) into this folder.
```

File: README.md [hunk 7]
```diff
@@ -114,9 +132,9 @@ There is currently no way of using this on AnkiMobile (iOS).
...
 
-        <details> <summary>Expected file structure <i>(click here)</i></summary>
+        <details> <summary>Expected file structure for people who learn Japanese <i>(click here)</i></summary>
```
Comment (MarvNC) [inline README.md:137]:
```suggestion
        <details> <summary>Expected file structure for Japanese audio <i>(click here)</i></summary>
```

File: README.md [hunk 8]
```diff
@@ -151,6 +169,33 @@ There is currently no way of using this on AnkiMobile (iOS).
 
         </details>
 
+    * For people who learn other language: change the name of folder from the second step to `forvo_files` and copy the entire folder _into_ `user_files` folder.
```
Comment (MarvNC) [inline README.md:172]:
```suggestion
    * For other languages, change the name of the folder from the second step to `forvo_files` and copy the entire folder _into_ the `user_files` folder.
```

File: README.md [hunk 9]
```diff
@@ -151,6 +169,33 @@ There is currently no way of using this on AnkiMobile (iOS).
...
+
+        <details> <summary>Example file structure for people who learn other language <i>(click here)</i></summary>
```
Comment (MarvNC) [inline README.md:174]:
```suggestion
        <details> <summary>Example file structure for other languages' audio <i>(click here)</i></summary>
```

**Review by MarvNC — CHANGES_REQUESTED** [review]

File: README.md [hunk 10]
```diff
@@ -88,14 +97,23 @@ There is currently no way of using this on AnkiMobile (iOS).
         >     ```
         >
         >     </details>
+        >
+        > -   **For people who learn other language**, use the magnet link below and select to download _only_ zip file of the language under the folder Forvo_pronunciation > export > mp3 > [iso 639 code of language that you want to learn.zip](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes):
```
Comment (MarvNC) [inline README.md:101]:
```suggestion
        > -   **For audio of other languages**, use the magnet link below containing Forvo audio and select the zip file of the language you want under the folder `Forvo_pronunciation > export > opus > [iso 639 code of language that you want to learn.zip](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes)`:
```

**Review by MarvNC — APPROVED** [review]
I think the overall structure of the readme might need a little more organization but that can come in a later PR.

**Review by jamesmaa — APPROVED** [review]

**Comment by Geniusssmit** [comment]
> I got Chinese audio files from forvo working through this add on but i had to move all the files to the same folder [forvo_files] where the japanese user folders also are. Especially since Chinese and even many other languages are now being supported by Yomitan, would it be possible to update this add on to have a separate source/folder for other languages like for chinese "zh forvo" or something like that? That way we can just specify in yomitan in the custom url json "language=zh" to only fetch the chinese files. Or at least can you add instructions for how to separate language folders for forvo in the add ons folder?  The way it is now, I need to specify all the chinese speakers with forvo usernames in the custom url.

Where can I download files for Mandarin Chinese? I you have them, please share :)

**Pull request merged**